cli/scorecard/inventory.go

```diff
@@ -29,49 +29,65 @@ import (
 // InventoryConfig manages a CAI inventory
 type InventoryConfig struct {
 	projectID        string
-	controlProjectID string
+	folderID   		 string
 	organizationID   string
 	bucketName       string
 	dirPath          string
 	readFromStdin    bool
+	refresh			 bool
```

Comment:
I don't think there's any need to store this on the config?

Reply:
removed in c317f93